src/python/pants/option/global_options.py

```diff
@@ -201,3 +201,5 @@ def register_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--force', type=bool, recursive=True,
```

Comment:
Yeah, so it'd be `./pants --cache-force ...` and `./pants test --cache-force ...` for global and local respectively. The 'cache' prefix seems odd, as a user I'm forcing execution, I don't really know about 'cache'. Another opinion would be good.